libobs/media-io/video-matrices.c

```diff
@@ -243,11 +243,14 @@ bool video_format_get_parameters(enum video_colorspace color_space,
 				 float range_min[3], float range_max[3])
 {
 	uint32_t bpc = 8;
+	PRAGMA_WARN_PUSH
+	PRAGMA_WARN_SWITCH
 	switch (color_space) {
 	case VIDEO_CS_2100_PQ:
 	case VIDEO_CS_2100_HLG:
 		bpc = 10;
 	}
+	PRAGMA_WARN_POP
```

Comment:
Instead of disabling the warning, I'd like to suggest using `default` condition and assigning `8`.
```c
        uint32_t bpc;
        switch (color_space) {
        case VIDEO_CS_2100_PQ:
        case VIDEO_CS_2100_HLG:
                bpc = 10;
                break;
        default:
                bpc = 8;
        }
```

Reply:
`default` is a no go there, it needs to be all cases in the future. But for now, in this PR it's pragma.